packages/util/src/bn/max.ts

```diff
@@ -20,12 +20,7 @@ import assert from '../assert';
  * ```
  */
 export default function max (
-  ...items: BN[]
+  ...items: BnList
 ): BN {
-  assert(
```

Comment:
Don't drop the assert, use both :)

Reply:
Yeah, my message was maybe not clear. Assert is useful, BNList type is optional